src/Controller/JobController.php

```diff
@@ -59,6 +59,14 @@ public function rss(): Response
         ]);
     }
 
+    #[Route('/entityTag', name: 'job_index_entity_tag', defaults: ['_format' => 'html'], methods: ['GET']), ]
```

Comment:
What is the purpose of this route?

Reply:
It's a temporary route for check the display the entity tag.